features/composition-events.yml

```
@@ -0,0 +1,10 @@
+name: Composition events
+description: The `compositionstart`, `compositionupdate` and `compositionend` events fire when text containing characters that might not be commonly available on keyboards are added. Examples of these characters are emoji, logograms in Asian languages, or input from speech recognition processors.
```

Optional suggestion: Seems like "input method editor" might be a key term here.

```suggestion
description: The `compositionstart`, `compositionupdate`, and `compositionend` events fire when an input method editor adds characters that might not be commonly available on keyboards. Examples of these characters are emoji, logograms in Asian languages, or input from speech recognition processors.
```